business_requirement/README.rst

```diff
@@ -0,0 +1,281 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: https://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+
+====================
+Business Requirement
+====================
+
+Introduction
+^^^^^^^^^^^^
+
+This module is part of a set ("Business Requirement") and provides the basic models for business requirement management and project time/cost estimation.
+
+|image7|
+
+.. |image7| image:: static/img/bus_req_tree.png
+   :width: 800 px
+   :alt: Business Requirement List view 
+
+
+The set comprises of multiple modules that can be used independently or not:
+
+=========================================== ====================================
```

Comment:
I think this a description for the README.md of the repository, not this module, and it's even automatically done by the module listing bot.